weaviate/tests/fixtures/weaviate_meta_api.txt

```diff
@@ -0,0 +1 @@
+{"hostname":"http://[::]:8080","modules":{},"version":"1.19.1"}
```

Comment:
Should this and the other api response fixture have "json" file extensions? for clarity